src/views/homepage/content-schema.ts

```diff
@@ -56,20 +56,18 @@ const merchandisingSchema = Joi.object({
 /**
  * Learn section
  */
-interface LearnSectionContent {
+interface CertificationsSectionContent {
 	heading: string
-	description: string[]
-	imageSrc: string
+	description: string
 	link: {
 		url: string
 		text: string
 	}
 	collectionSlugs: string[]
 }
-const learnSectionSchema = Joi.object({
+const certificationsSectionSchema = Joi.object({
```

Comment:
Have updated the homepage content schema slightly - `imageSrc` is no longer used, and have simplified `description` to be a string (we only want one paragraph of text in our new layout).

📌 Seems like a good place to eventually swap in `zod` for `joi`, to benefit from inferred types. That felt very optimize-y though, so figured better to focus on feature work for now. Happy to go on a `zod` side quest if nothing else feels higher priority 👍

Reply:
I think doing the zod work as a later effort to fully rip out joi makes sense!

Reply:
[made a ticket here for that work](https://app.asana.com/0/1202097197789424/1203643020909876)